README.adoc

```diff
@@ -1 +1,53 @@
-= bisq network roles
+= Bisq Network Roles
+
+This repository gives you a transparent overview about all current roles within the Bisq Network. It provides the viewer with details about the purpose, motivation, privileges, requirements and responsibilities of all roles. This will ensure clarity about ones own role, help identifying other roles and facitilitate the compensation process.
+
+The **owner of the role** is responsible for **creating and maintaining** of his/her **role description**. 
+
+== How to Create a New Role?
+
+1. Create **New File**
+1. **Name** Your File according to the defined syntax (eg. "role-maintainer.adoc")
+1. **Edit** Your Role according to the https://github.com/bisq-network/roles/blob/master/TEMPLATE.adoc[Template]
+1. **Propose** New File
+1. You're done? Then you are ready to Create a **Pull Request**
+1. Wait for Confirmation
```

Comment:
We can probably shorten this series of steps to:

## How to create a new role

1. Discuss and make sure there is consensus about the need for and purpose of this role in the #roles channel or elsewhere
2. Create a GitHub issue representing that will be used to track this role (see other [role issues](https://github.com/bisq-network/roles/issues) for examples)
3. Create and edit a 'specification' document for this role based on `ROLE-SPEC-TEMPLATE.adoc`. See the existing list of role specs and follow the naming convention.
4. Submit the new file as a pull request. Be sure to mention the role issue ID, e.g. #28 somewhere in the pull request title or description to ensure that the pull request shows up in the timeline of the role issue.
5. The roles repository maintainer will review your new role spec, provide any feedback, and merge it when ready.